### ISSUE TITLE
[Due for payment 2026-01-30] Screen Readers: Settings: The text block is announced without displayed details

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77452]
## Action Performed:
Precondition: The User should be logged in

1. Using Android TalkBack or iOS VoiceOver, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe to the text block at the top of the screen
4. Note how it is announced.

## Expected Result:
The expected result is a single, complete announcement upon focus: "User Profile for [Full Name]. Email: [Email Address]."

## Actual Result:
The text block at the top of the screen is announced without displayed details; it is announced as 'Profile'.

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Use the accessibilityLabel to announce all displayed details. e.g.:
- 'User Profile for [Full Name]. Email: [Email Address]'
https://appt.org/en/docs/react-native/samples/accessibility-label

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

![Image](https://github.com/user-attachments/assets/4f4e566f-baed-409e-a6ed-215c12dbcf28)

https://github.com/user-attachments/assets/9a69e396-5136-44b2-8363-04b9761a0ea3

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes 
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006014
**Issue reported by:** NA
**Slack conversation:** NA
